classes/Pods.php

```diff
@@ -1035,89 +1076,73 @@ public function field( $name, $single = null, $raw = false ) {
 					}
 
 					$object_field_found = true;
-				}
-			}
 
-			if ( ! $object_field_found ) {
-				foreach ( $image_fields as $image_field ) {
-					if ( isset( $this->fields[ $image_field ] ) ) {
-						// Skip any names that are registered fields within this Pod.
-						continue;
-					}
-					if (
-						$image_field === $params->name ||
-						0 === strpos( $params->name, $image_field . '.' )
-					) {
-						// Is it a URL request?
-						$url = '_url' === substr( $image_field, -4 );
-						if ( $url ) {
-							$image_field = substr( $image_field, 0, -4 );
-						}
-
-						// Results in an empty array if no traversal names are passed.
-						$traverse_names = explode( '.', $params->name );
-						array_shift( $traverse_names );
-
-						$attachment_id = 0;
-						switch ( $image_field ) {
-							case 'post_thumbnail':
-								$attachment_id = get_post_thumbnail_id( $this->id() );
-								break;
-							case 'image_attachment':
-								if ( isset( $traverse_names[0] ) ) {
-									$attachment_id = $traverse_names[0];
-									array_shift( $traverse_names );
-								}
-								break;
-						}
+				} elseif( in_array( $first_field, $image_fields, true ) ) {
```

Comment:
* :no_entry_sign: There must be a space between elseif and the first parenthesis '('
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after ELSEIF keyword; 0 found

<sub>hash: a19ec08b80090c11e1d444ac5db212fc</sub>